### DISCUSSION TITLE
What's the meaning of the question marks next to prices when price checking an item?

### DISCUSSION THREAD

**Discussion by baranelitez** [SnosMe/awakened-poe-trade#664] [Q&A] [answered]
Hi. What's the meaning of the question marks next to prices? I see something like that is mentioned in https://github.com/SnosMe/awakened-poe-trade/releases/tag/v2.15.0 but I fail to understand what `Items priced using Stash Tab Name now have "?" after the price` means. I'd be grateful if you could clarify this for me. Thanks!

https://pasteboard.co/NgvGPbs3md9u.png here's a screenshot where the q-marks appear.

**Comment by freezerain** [comment, marked as answer]
I think thats mean that item is not priced individualy but just sits inside stash "all 69 chaos". 
Practically speaking that mean you can try ask seller to lower the price

**Reply by SnosMe** [reply]
That's correct.
The intention here is to make you aware that price is likely not real and is only so high because the item has been thrown in dump tab.
